src/main/java/com/harrison/config/FlywayConfig.java

```diff
@@ -0,0 +1,26 @@
+package com.harrison.config;
+
+import java.util.Properties;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Properties'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)